tests/unit/types/test_routing_graph.py

```diff
@@ -110,7 +110,8 @@ def test_topological_sorting():
     assert topological_sorting[2] in ['pod1', 'pod2', 'pod3']
     assert topological_sorting[3] in ['pod2', 'pod3']
     assert topological_sorting[4] == 'pod4'
-    assert topological_sorting == ['pod0', 'pod2', 'pod1', 'pod3', 'pod4']
+    # TODO FIX THIS, topological_sorting get random result each time!
+    # assert topological_sorting == ['pod0', 'pod2', 'pod1', 'pod3', 'pod4']
```

Comment:
then assert the sets equality

Reply:
but we're testing the `sorting`, it should be ordered, right? I'd say let's make it as todo and debug in a seperate ticket

Reply:
ok

Reply:
now I see the test it is not coherent with the rest of the asseets it only makes sense if u assert the set